### PR TITLE
Harden HTTPS

### PR DESCRIPTION
**Pull request by gagou9**
Adding DH Prime, using "intermediate compatibility" from Mozilla, enabling HSTS and OCSP stappling.

### CODE REVIEW

File: chef/site-cookbooks/wca/templates/wca_https.conf.erb [hunk 1]
```diff
@@ -1,15 +1,56 @@
+### 
+# Following this :
+# https://raymii.org/s/tutorials/Strong_SSL_Security_On_nginx.html
```
Comment (lgarron) [inline chef/site-cookbooks/wca/templates/wca_https.conf.erb]: This guide looks good, although I usually use https://mozilla.github.io/server-side-tls/ssl-config-generator/ (which stays up to date).

**Comment by jfly** [comment]
LGTM! @lgarron, assigning to you to merge up if you see fit.

**Comment by jfly** [comment]
Thanks @gagou9! This looks great!

File: chef/site-cookbooks/wca/templates/wca_https.conf.erb (hunk 1 above)
Comment (gagou9) [inline chef/site-cookbooks/wca/templates/wca_https.conf.erb]: > This guide looks good, although I usually use https://mozilla.github.io/server-side-tls/ssl-config-generator/ (which stays up to date).

Would you like me to change the link in the file? (I'm wondering as your
sentence is not that explicit ;) )

Comment (lgarron) [inline chef/site-cookbooks/wca/templates/wca_https.conf.erb]: Well, I would, but only if you were following the Mozilla generator. ;-)

**Comment by lgarron** [comment]
Looks good; thanks for generating the DH params.

**Pull request merged**